### PR TITLE
feat: log when not response.ok

### PR DESCRIPTION
**Pull request by danielgent**
Surely we want to log responses when things go wrong? I figure we can release this to dev and see what happens :see_no_evil: 

*also* I could add extra assertions to lots of our existing hono tests and check that we're populating the logs table in SQLite

### CODE REVIEW

File: src/tsoa-middlewares/logger.ts [hunk 1]
```diff
@@ -37,33 +37,31 @@ export function loggerMiddleware(logType: string, description?: string) {
       console.error(e);
     }
 
-    if (response.ok) {
```
Comment (danielgent) [inline src/tsoa-middlewares/logger.ts:40]: what was the reason for this @markusahlstrand ? I imagine `response.ok` will be false in the event that `next()` throws a HTTP exception? :thinking:

**Comment by markusahlstrand** [comment]
I think we need to add some try catch magic in here? And make sure it's consistent with non-ok responses?

**Review by markusahlstrand — COMMENTED** [review]
Gah.. added the comment in the comments. Not sure how these things are connected.

**Review by markusahlstrand — APPROVED** [review]
Looks good!

**Pull request merged**